ssd1351/src/main/java/com/example/upm/androidthings/driversamples/SpiDisplayActivity.java

```diff
@@ -106,6 +114,7 @@ public void run() {
                 display.setCursor((short)5, (short)70);
                 display.print("IoTDK");
                 display.refresh();
+                updateUI("Intel IoTDK");
```

Comment:
same message as above. Change to "IoTDK" without "Intel" as in the display.print?